{"name": "Тест получения заказа курьером с несуществующим id курьера", "status": "passed", "description": "Тест проверяет появление ошибки при получении заказа курьером с несуществующим id курьера", "steps": [{"name": "Получаем id заказа", "status": "passed", "steps": [{"name": "Регистрируем новый заказ", "status": "passed", "start": 1721315331378, "stop": 1721315331518}], "start": 1721315331378, "stop": 1721315331518}], "start": 1721315331378, "stop": 1721315331652, "uuid": "6f2dff27-3f70-4c81-ba1f-754d1a04e128", "historyId": "7340bb1811a392153b914aaffb7f586b", "testCaseId": "7340bb1811a392153b914aaffb7f586b", "fullName": "tests.test_accept_order.TestAcceptOrder#test_accept_order_invalid_courier_id", "labels": [{"name": "feature", "value": "Тест принятых заказов"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_accept_order"}, {"name": "subSuite", "value": "TestAcceptOrder"}, {"name": "host", "value": "DESKTOP-L3DLL49"}, {"name": "thread", "value": "8176-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_accept_order"}]}